test/aws-xray-sdk/tc_context.rb

```diff
@@ -39,4 +40,10 @@ def test_log_error
     context.context_missing = 'LOG_ERROR'
     refute context.current_entity
   end
+
+  def test_default_context_missing
+    context = XRay::DefaultContext.new
+    context.clear!
+    refute context.current_entity
```

Comment:
Is there an easy way to see if a log was added here?

Reply:
Good callout. The XRay Ruby SDKs don't test the output of the logger anywhere else unfortunately, it won't be a quick change. This can be tackled in another enhancement.

Reply:
Sounds good. Feel free to merge this then.